File: lecture002/fibonacci.java
import java.util.Scanner;
public class fibonacci{
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        int n = scn.nextInt();
        int a=0;
        int b=1;
        int sum=0;
        if (n>=1){
            System.out.println("Fibonacci series:"+a);
            if(n==1)
            return;
        }
        if (n>=2){
            System.out.println("Fibonacci series:"+b);
            if(n==2)
            return; 
        }
        for(int i=3;i<=n;i++){
            sum=a+b;
            a=b;
            b=sum; 
            System.out.println(sum+" ");
        }
        //System.out.println();
        //System.out.println(sum); 
    }
}

File: lecture002/fibonacci.cpp
#include<iostream>
using namespace std;
int main()
{
    int a,b,sum,n;
    cin>>n;
    a=0;
    b=1;
    if(n==1)
        cout<<a;
     else if(n==2)
        cout<<b;
     else{
        for(int i=3;i<=n;i++){
        sum=a+b;
        a=b;
        b=sum;
        if(i==n)
            cout<<sum;
        }
    }
}